src/rosconsole/rosconsole.cpp

```diff
@@ -63,6 +63,8 @@ void shutdown();
 
 void register_appender(LogAppender* appender);
 
+void deregister_appender();
```

Comment:
The implementation of `deregister_appender` might need to know which appender should be deregistered so I think this function needs to take the same argument as `register_appender`.

While that might not be necessary for the existing implementation it could be implementation using more than one appender.